GetRepoJSON/getRepoJSON.py

```diff
@@ -0,0 +1,52 @@
+# repos.py
+
+# A demo script that demonstrates api calls /user
+
+
+
+from time import sleep
+def delayserver():
+    sleep(0.01)
+    return
+
+
+import sys
+sys.path.append('..')
+from allspice_proto.allspice_proto import AllSpice_Proto
```

Comment:
```suggestion
from allspice_proto.allspice_proto import AllspiceProto
```

Convention in Python is that class names are camel case.

If I'm being pedantic, "allspice" is one word, which is why the S should be lowercase.  Everyone agrees that in snake case, it's `allspice_proto`, but I haven't been able to get people to agree on what the same thing is in camel case.

Reply:
I changed all code references to "allspice". I left non-code branded references to "AllSpice".